Src/library/docking/implementation/docking/sd_docking_manager_command.e

```diff
@@ -17,31 +17,29 @@ inherit
 	SD_ACCESS
 
 	SD_DOCKING_MANAGER_HOLDER
-		redefine
-			set_docking_manager
-		end
 
 create
 	make
 
 feature {NONE}  -- Initlization
 
-	make
-			-- Creation method
+	make (a_docking_manager: SD_DOCKING_MANAGER)
+			-- Associate new object with `a_docking_manager'.
 		do
+			docking_manager := a_docking_manager
 			create locked_windows.make (10)
```

Comment:
Looks to me that we should be using an ARRAYED_LIST for `locked_windows`. Not required now, but it would slightly simplify the code now.